app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal:
+    def __init__(self, name, appetite, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+    def __init__(self, name, appetite=3, is_hungry=True):
+        super().__init__(name, appetite, is_hungry)
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
```

Comment:
you passed the same argumenst into father constuctor with function super, shoudn`t reassign them below. If you want set default values you can do it when call method super().__init__(variable=default_value)

Reply:
All changes provided. You helped me to understand the super() method, thanks!